api/src/AppBundle/Entity/CourtOrderAddress.php

```diff
@@ -0,0 +1,173 @@
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $addressLine3 = '';
+
+    /**
+     * @var string|null
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $town = '';
+
+    /**
+     * @var string|null
+     * @ORM\Column(type="string", nullable=true)
+     */
+    private $county = '';
+
+    /**
+     * @var string|null
+     * @ORM\Column(type="string", length=10, nullable=true)
+     */
+    private $postcode = '';
+
+    /**
+     * @var string|null
+     * @Assert\Country
+     * @ORM\Column(type="string", length=2, nullable=true)
+     */
+    private $country = '';
+
+    /**
+     * @var CourtOrderDeputy
+     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\CourtOrderDeputy", inversedBy="addresses", cascade={"persist"})
+     * @ORM\JoinColumn(name="court_order_deputy_id", referencedColumnName="id", onDelete="CASCADE")
+     */
+    private $deputy;
+
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    public function getAddressLine1(): ?string
+    {
+        return $this->addressLine1;
+    }
+
+    public function getAddressLine2(): ?string
+    {
+        return $this->addressLine2;
+    }
+
+    public function getAddressLine3(): ?string
+    {
+        return $this->addressLine3;
+    }
+
+    public function getTown(): ?string
+    {
+        return $this->town;
+    }
+
+    public function getCounty(): ?string
+    {
+        return $this->county;
+    }
+
+    public function getPostcode(): ?string
+    {
+        return $this->postcode;
+    }
+
+    public function getCountry(): ?string
+    {
+        return $this->country;
+    }
+
+    public function getDeputy(): CourtOrderDeputy
+    {
+        return $this->deputy;
+    }
+
+    public function setAddressLine1(?string $addressLine1): CourtOrderAddress
+    {
+        $this->addressLine1 = $addressLine1;
+
+        return $this;
+    }
+
+    public function setAddressLine2(?string $addressLine2): CourtOrderAddress
+    {
+        $this->addressLine2 = $addressLine2;
+
+        return $this;
+    }
+
+    public function setAddressLine3(?string $addressLine3): CourtOrderAddress
+    {
+        $this->addressLine3 = $addressLine3;
+
+        return $this;
+    }
+
+    public function setTown(?string $town): CourtOrderAddress
+    {
+        $this->town = $town;
+
+        return $this;
+    }
+
+    public function setCounty(?string $county): CourtOrderAddress
+    {
+        $this->county = $county;
+
+        return $this;
+    }
+
+    public function setPostcode(?string $postcode): CourtOrderAddress
+    {
+        $this->postcode = $postcode;
+
+        return $this;
+    }
+
+    public function setCountry(?string $country): CourtOrderAddress
+    {
+        $this->country = $country;
+
+        return $this;
+    }
+
+    public function setDeputy(CourtOrderDeputy $deputy): CourtOrderAddress
+    {
+        $this->deputy = $deputy;
+
+        return $this;
+    }
+}
```

Comment:
Two thoughts about this entity:
* It's a CourtOrderAddress yet it has reference to a CourtOrderDeputy, not a CourtOrder. If the reference is correct, should it be a CourtOrderDeputyAddress, else should we update the reference to a $courtOrder instead of a $deputy?

* That aside, do we want any relation at all? Besides from that relation it's just a generic Address object, which could have a many-to-many relationship with deputies/organisations if we removed that mapping to an entity?

Reply:
As discussed on Meet, will turn this into `CourtOrderDeputyAddress` (and the corresponding DTO). It'll remain a one-to-many with CourtOrderDeputy to avoid collisions and data creep.